docs/content/reference/install-configuration/observability/metrics/metrics.md

```diff
@@ -0,0 +1,617 @@
+---
+title: "Traefik Metrics Overview"
+description: "Traefik Proxy supports these metrics backend systems: Datadog, InfluxDB 2.X, Prometheus, and StatsD. Read the full documentation to get started."
+---
+
+# Metrics
+
+Traefik provides metrics in the [OpenTelemetry](#open-telemetry) format as well as the following vendor specific backends:
+
+- [Datadog](#datadog)
+- [InfluxDB2](#influxdb-v2)
+- [Prometheus](#prometheus)
+- [StatsD](#statsd)
+
+Traefik Proxy hosts an official Grafana dashboard for both [on-premises](https://grafana.com/grafana/dashboards/17346)
+and [Kubernetes](https://grafana.com/grafana/dashboards/17347) deployments.
+
+---
+
+## Open Telemetry
+
+### Helm Chart Configuration
+
+Traefik can be configured to provide metrics in the OpenTelemetry format using the Helm Chart values.
+To know more about the Helm Chart options, refer to the [Helm Chart](https://github.com/traefik/traefik-helm-chart/blob/master/traefik/VALUES.md) (Find options `metrics.otlp`).
+
+### Configuration Example
+
+To enable the OpenTelemetry metrics:
+
+```yaml tab="File (YAML)"
+metrics:
+  otlp:
```

Comment:
This is not the correct configuration, this is the Helm one, see https://doc.traefik.io/traefik/observability/metrics/opentelemetry/#opentelemetry